contracts/dnssec-oracle/BytesUtils.sol

```diff
@@ -76,10 +85,10 @@ library BytesUtils {
             if (a != b) {
                 // Mask out irrelevant bytes and check again
                 uint256 mask;
-                if (shortest > 32) {
+                if (shortest - idx >= 32) {
                     mask = type(uint256).max;
                 } else {
-                    mask = ~(2**(8 * (32 - shortest + idx)) - 1);
+                    mask = ~(2 ** (8 * (idx + 32 - shortest)) - 1);
```

Comment:
Is there any difference by changing from `32 - shortest + idx` to `idx + 32 - shortest`?

Reply:
See the original bug report - there's overflow issues with the previous ordering.